src/pages/Login/Login.js

```diff
@@ -1,7 +1,297 @@
-import React from 'react';
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { REST_API_KEY, REDIRECT_URI } from './KakaoLoginData';
+import { useNavigate } from 'react-router-dom';
+import Modal from 'react-modal';
 
 const Login = () => {
-  return <div>login</div>;
+  const navigate = useNavigate();
+  const [modalIsOpen, setIsOpen] = React.useState(false);
+  const [checkedOne, setCheckedOne] = useState(false);
+  const [checkedSecond, setCheckedSecond] = useState(false);
+  const props = checkedOne && checkedSecond ? { isValid: true } : {};
```

Comment:
```suggestion
  const props = checkedOne && checkedSecond;
```
이렇게 관리하면 어떨까요?
그리고 변수명을 props 보다는 의미가 더 명확한 변수명을 사용해 주세요!

Reply:
isValid 부분을 지우니까 버튼 색깔이 안 바뀌어서 일단 변수명만 바꿔두었습니다!